### DISCUSSION TITLE
How to upload some files to a specific directory?

### DISCUSSION THREAD

**Discussion by miaogaolin** [ObsidianPublisher/obsidian-github-publisher#264] [Q&A — Plugin]
How do I configure it, thank you very much.

This is my configuration:
![image](https://github.com/ObsidianPublisher/obsidian-github-publisher/assets/57740293/02970ff5-489f-4162-bdc5-0e93b8f25bcf)

By default all files are uploaded to the content/posts directory, but individual files I want to upload to the content directory, how do I configure this.

**Comment by Mara-Li** [comment]
There a two solution:
- Using frontmatter option, where you set by default like that : 
```json
{
"upload": {
    "behavior": "yaml",
    "defaultName": "content/posts",
    "rootFolder": "",
    "yamlFolderKey": "directory",
   }
}
```
So you just need to use `directory: content`
- Using the frontmatter key `path` that overrides everything configured (so you can use `path: content`
- Using the path regex replace, but you need to have a specific path for theses files.
